luminoth/train.py

```diff
@@ -232,7 +229,7 @@ def train(*args, **kwargs):
 
     # If cluster information is empty or TF_CONFIG is not available, run local
     if job_name is None or task_index is None:
-        return run(*args, **kwargs)
+        return run(config_files, override_params)
```

Comment:
Maybe all the configs could be resolved/merged outside `run()`, so that function only does its thing?